docs/TodayWidget.md

```diff
@@ -0,0 +1,27 @@
+---
+id: today-widget
+title: Today Widget
+layout: docs
+category: Guides (iOS)
+permalink: docs/today-widget.html
+next: native-modules-android
+previous: building-for-apple-tv
+---
+
+The memory limit for a today widget is 16 MB. Unfortunately, today widget implementations using React Native work unreliably because the memory usage tends to be too high. Running builds in the simulator works for development purposes, but running builds in the memory-limited environment of a device's notification view may not always work. The memory usage is too high, yielding the message 'Unable to Load' in the today widget's view.
```

Comment:
I feel like this could use an intro to app extensions and the Today Widget, as not everyone reading this may be familiar with classic iOS development.